CppCoreGuidelines.md

```diff
@@ -7152,16 +7152,16 @@ Some of these expressions are unconditionally bad (e.g., they rely on undefined
 
 **Example**:
 
-	x=k*y+z;	// OK
+	x=k*y+z;              // OK
 
-	auto t1 = k*y;	// bad: unnecessarily verbose
+	auto t1 = k*y;        // bad: unnecessarily verbose
 	x = t1+z;
 
-	if(0<=x && x<max) // OK
+	if(0<=x && x<max) {}  // OK
```

Comment:
We do like the general idea.  We just need to find a balance between human communication and compiler happiness :-)